cogs/msg.py

```diff
@@ -74,7 +76,22 @@ async def on_message(self, message):
         if len(result) == 0:
             return
 
-        await message.channel.send(result[0][1])
+        text = result[0][1]
+
+        def isUrl(url):
+            try:
+                result = urlparse(url)
+                return all([result.scheme, result.netloc])
+            except:
+                return False
+
+        rest = message.clean_content[search.end():].strip()
```

Comment:
I feel like we should make a specific syntax for this, that can be built into the regex. Or just make it a command, `!msg memegen <shorthand> <text>` instead of weirdly overloading the already implicit behaviour?

Reply:
I chose `!msg caption` to not collide with the enable/disable commands